user-interface/6001-SORD-submit_orders.md

```diff
@@ -0,0 +1,139 @@
+# Submit order
+As a user I want change my exposure on a market (e.g. open a position, increase or decrease my open volume), I want to submit an order with instructions for how my order should be executed so I have some control over the price that I get, as well as if when/my order should stay on the book. See [specs about orders](../protocol#orders) for more info.
+
+When populating a deal ticket I...
```

Comment:
Something about deal tickets being market specific, i.e. remembering prices and sizes for each market separately rather than whatever was there before you swtiched market.

Reply:
resolved in https://github.com/vegaprotocol/specs/pull/1181/commits/efc4690a9777cefcd2166cbeb19056506a9718c8